packages/editor/src/components/alignment-toolbar/index.js

```diff
@@ -17,37 +17,37 @@ import { compose } from '@wordpress/compose';
  */
 import { withBlockEditContext } from '../block-edit/context';
 
-const ALIGNMENT_CONTROLS = [
+const DEFAULT_ALIGNMENT_CONTROLS = [
 	{
 		icon: 'editor-alignleft',
-		title: __( 'Align left' ),
+		title: __( 'Align text left' ),
 		align: 'left',
 	},
 	{
 		icon: 'editor-aligncenter',
-		title: __( 'Align center' ),
+		title: __( 'Align text center' ),
 		align: 'center',
 	},
 	{
 		icon: 'editor-alignright',
-		title: __( 'Align right' ),
+		title: __( 'Align text right' ),
 		align: 'right',
 	},
 ];
 
-export function AlignmentToolbar( { isCollapsed, value, onChange } ) {
+export function AlignmentToolbar( { isCollapsed, value, onChange, alignmentControls = DEFAULT_ALIGNMENT_CONTROLS } ) {
```

Comment:
This change is not needed anymore right? can we remove it?

Reply:
It is not needed but I liked this change, it allows a block for example to only offer right or left alignment, or to change labels in a specific case, e.g: instead of text naming the actual field "Author name align left" that may be very useful for a11y.